pyproject.toml

```diff
@@ -0,0 +1,38 @@
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
```

Comment:
:+1:

Reply:
I just followed your lead 🙂

Reply:
So you used https://github.com/JeanChristopheMorinPerso/otio-fcpxxml-adapter/blob/initial_structure/pyproject.toml as a reference?

Reply:
Yes.

Reply:
Great, I'm happy that it was useful!

Reply:
Added a segment on building packages.